index.html

```diff
@@ -18,7 +18,7 @@
       }
 
       #container {
-        background-color: pink;
+        background-color: rgb(109, 156, 242);
```

Comment:
love to see you having fun with colors! It certainly makes it a more personalized game!